docusaurus/docs/Android/04-compose/03-channel-components/03-channel-list.mdx

```diff
@@ -13,9 +13,9 @@ The **bound** version of the list uses the **stateless** list internally. That w
 
 Based on the provided state, the component shows the following UI:
 
-* `LoadingView`: If we're loading the initial data.
-* `EmptyView`: If loading is done but there is no data to show.
-* The list of channels: Channel items with various actions like item taps and long taps and pagination.
+* Loading view if we're loading the initial data.
```

Comment:
We shouldn't be using `View` in compose. Component or UI should be better to replace. Or in this case Indicator.